From d293f07659bdd7a855104f78486b629299ee6ed3 Mon Sep 17 00:00:00 2001
From: Murala S Swamy <ssmurala@habana.ai>
Date: Sat, 31 May 2025 12:32:46 +0300
Subject: [PATCH] [JSW-808] jgs: build palad boot loader

This patch enables palad boot loader build
for jgs, to start verification with stand alone
applications.

Change-Id: I8437cde048327983af06ad6826be2b710ebe9967
---

diff --git a/fs/jgs/address_space.h b/fs/jgs/address_space.h
index 3826b96..d62df51 100644
--- a/fs/jgs/address_space.h
+++ b/fs/jgs/address_space.h
@@ -15,10 +15,11 @@
  #define HL_CFG_BASE			0x80000000
  #define CFG_ARC_REGION_BASE		0xF0000000
 
- #define IO0_PCIE_FW_SRAM_BASE		(mmIO_0_CFG_HALF_0_FW_MEM_PCIE_FW_SRAM_BASE +\
+ #define IO0_PCIE_FW_SRAM_BASE		(mmIO_0_CFG_FW_MEM_PCIE_FW_BASE +\
 					  mmPCIE_FW_SRAM_SRAM_LOCATIONS)
- #define IO0_PCIE_FW_CSR_BASE		(mmIO_0_CFG_HALF_0_FW_MEM_PCIE_FW_SRAM_BASE +\
+ #define IO0_PCIE_FW_CSR_BASE		(mmIO_0_CFG_FW_MEM_PCIE_FW_BASE +\
 					  mmPCIE_FW_SRAM_CSR_LOCATIONS)
+
  #define IO0_FW_SRAM_BASE		mmIO_0_CFG_FW_MEM_FW_SRAM_BASE
  #define IO0_SP_SRAM_BASE		mmIO_0_CFG_SP_RAM_BASE
  #define IO0_RST_ICCM_BASE		mmIO_0_CFG_FW_MEM_RST_ARC_ICCM_BASE
@@ -32,8 +33,8 @@
  #define IO0_MUC_ICCM_BASE		mmIO_0_CFG_MEDIA_0_ICCM_BASE
  #define IO0_MUC_DCCM_BASE		mmIO_0_CFG_MEDIA_0_DCCM_BASE
 
- #define COMP0_RST_ICCM_BASE		mmCOMPUTE_0_CFG_FW_MEM_RST_ARC_ICCM_BASE
- #define COMP0_RST_DCCM_BASE		mmCOMPUTE_0_CFG_FW_MEM_RST_ARC_DCCM_BASE
+ #define COMP0_PM_ICCM_BASE		mmCOMPUTE_0_CFG_FW_MEM_PM_ARC_ICCM_BASE
+ #define COMP0_PM_DCCM_BASE		mmCOMPUTE_0_CFG_FW_MEM_PM_ARC_DCCM_BASE
  #define COMP0_MNG_ICCM_BASE		mmCOMPUTE_0_CFG_FW_MEM_MNG_ARC_ICCM_BASE
  #define COMP0_MNG_DCCM_BASE		mmCOMPUTE_0_CFG_FW_MEM_MNG_ARC_DCCM_BASE
 
@@ -55,8 +56,8 @@
  #define IO0_MUC_ICCM_FULL_BASE		(FW_MEM_FULL_BASE + IO0_MUC_ICCM_BASE)
  #define IO0_MUC_DCCM_FULL_BASE		(FW_MEM_FULL_BASE + IO0_MUC_DCCM_BASE)
 
- #define COMP0_RST_ICCM_FULL_BASE	(FW_MEM_FULL_BASE + COMP0_RST_ICCM_BASE)
- #define COMP0_RST_DCCM_FULL_BASE	(FW_MEM_FULL_BASE + COMP0_RST_DCCM_BASE)
+ #define COMP0_RST_ICCM_FULL_BASE	(FW_MEM_FULL_BASE + COMP0_PM_ICCM_BASE)
+ #define COMP0_RST_DCCM_FULL_BASE	(FW_MEM_FULL_BASE + COMP0_PM_DCCM_BASE)
  #define COMP0_MNG_ICCM_FULL_BASE	(FW_MEM_FULL_BASE + COMP0_MNG_ICCM_BASE)
  #define COMP0_MNG_DCCM_FULL_BASE	(FW_MEM_FULL_BASE + COMP0_MNG_DCCM_BASE)
 
